packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```diff
@@ -53,6 +53,22 @@ This package contains documentation for rubygem-%{gem_name}.
 %build
 
 %install
+for i in ansible ansible_galaxy; do
+if [ -d %{foreman_proxy_dir}/.$i ]; then
+  mv %{foreman_proxy_dir}/.$i %{_localstatedir}/lib/foreman-proxy/$i
```

Comment:
Moving this should happen in ```%pre``` because that step happens on the actual system. ```%install``` happens while building in a chroot. Only adding the symlink should happen in ```%install```